packages/babel-plugin-transform-flow-strip-types/src/index.js

```diff
@@ -19,6 +19,22 @@ export default function ({ types: t }) {
         }
       },
 
+      ImportDeclaration(path) {
+        if (!path.node.specifiers.length) return;
```

Comment:
Apparently so atm, `import {} from "babel-traverse";` parses.. but it turns into `import "babel-traverse";` basically